go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/section-io/module-metrics
 go 1.13
```

Comment:
Worth updating the Golang version while we are at it.

Reply:
Possibly a good time, but really only a secondary concern.  Can be done fairly quickly once we've got end-to-end working.